react/Input/Readme.md

```diff
@@ -34,6 +34,30 @@
 <Input placeholder="I'm full width" fullwidth />
 ```
 
+### inputRef ( set focus / blur / ... programmatically)
+
+```
```

Comment:
```suggestion
```If you need to programmatically access the underlying `<input />` for example to give focus or move the caret, you can use the `innerRef` prop, that is passed to the `ref` property of the `<input />`.
```

Reply:
is it better now ?